test/assets/utils.js

```diff
@@ -125,6 +150,6 @@ module.exports = {
     logger: mainLogger,
     createRoom: createRoom,
     createSocket: createSocket,
-
+    getRoom: getRoomProject,
```

Comment:
It might not be a bad idea to just replace the current createRoom with this function and update the tests accordingly